public/app/features/dashboard/components/DashboardLoading/DashboardFailed.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+import { css } from 'emotion';
+import { Alert, useStyles } from '@grafana/ui';
+import { GrafanaTheme } from '@grafana/data';
+import { DashboardInitError, AppNotificationSeverity } from 'app/types';
+import { getMessageFromError } from 'app/core/utils/errors';
+
+export interface Props {
+  initError?: DashboardInitError;
+}
+
+export const DashboardFailed: FC<Props> = ({ initError }) => {
```

Comment:
```suggestion
export const DashboardFailed = ({ initError }: Props) => {
```
We decided on frontend weekly to stop using the FC type,

Reply:
yeah...true.
would make changes to this